guideline.md

```diff
@@ -0,0 +1,126 @@
+# Pyladies Germany PySV Grant Program
+
+The annual €5000 grant program from the PySV for PyLadies aims to support the activities and growth of PyLadies chapters in Germany. It is essential to ensure that these funds are used transparently, therefore we have devised the following guidelines. The grant will be issued to PyLadies each year after the PyCon DE conference (circa June), once assigned the funds are available for disposal by the PyLadies using the following guidelines. Money not used within the year will stay on the PyLadies account and can continue to be used.
+
+**Allowable expenses:**
+- Community events: this includes room rentals, refreshments, and similar costs.
+- Promotional materials: such as stickers, banners, T-shirts, and other marketing materials.
+- Software tools: Subscription-based tools or one-time software purchases that benefit the chapter (these may also be shared across chapters).
```

Comment:
maybe we can use what PyLadies Berlin is using to share passwords and so on :)

Reply:
yeh good shout, we could also have within this repo a list of those tools (and current state of them) then link that here? For Passwords we are using Lastpass and tbh I only "kept" the subscription because I forgot to cancel it and was then charged for it :(

Reply:
I made a separate readme to track shared tools and linked it here